README.md

```diff
@@ -87,6 +98,11 @@ npm test
 
 Or alternatively just run `npm run lint` or `npm run test:unit` individually. If you're devving and you've got eslint errors you need to fix, run `npm run lint:fix`.
 
+To run a single test you can run:
```

Comment:
I think this section "running a single test" makes sense to live in frontend-testing-base rather than here